packages/test/test-end-to-end-tests/src/test/summaries.spec.ts

```diff
@@ -307,6 +325,73 @@ describeNoCompat("Summaries", (getTestObjectProvider) => {
 		);
 	});
 
+	itExpects(
+		"full initialization of data object should not happen by default",
+		[
+			{
+				eventName: "fluid:telemetry:Container:Request_cancel",
+				error: "Non interactive/summarizer client's data object should not be initialized",
+			},
+		],
+		async () => {
+			const TestDataObjectType1 = "@fluid-example/test-dataStore1";
+			const dataStoreFactory1 = new DataObjectFactory(
+				TestDataObjectType1,
```

Comment:
nit: `TestDataObjectType1` is only used in one place. You can directly use "@fluid-example/test-dataStore1" here avoiding creating extra const variables.
In general, its a good idea to prevent additional variables if not required.